tests/test_reco_diversity.py

```diff
@@ -182,3 +187,77 @@ def test_inter_list_diversity_metric_euclidean(self):
 
         self.assertEqual(round(results['inter-list diversity'], 4), 1.4142)
         self.assertEqual(results['support'], 2)
+
+    def test_intra_list_diversity_usage(self):
+        metric = DiversityRecoMetrics.IntraListDiversity(self.item_features, click_column='score', k=4)
```

Comment:
Nice tests! One thing might be updated is to capture the warning message if any, to ease the tests. That is also an update I would like to have for inter-list.